python/graphscope/client/session.py

```diff
@@ -165,6 +167,9 @@ def __init__(
         k8s_volumes=gs_config.k8s_volumes,
         k8s_waiting_for_delete=gs_config.k8s_waiting_for_delete,
         timeout_seconds=gs_config.timeout_seconds,
+        access_key_id = gs_config.access_key_id,
+        access_key_secret = gs_config.access_key_secret,
+        region = gs_config.region,
```

Comment:
As discussed, are we provide a script to launch AWS/Aliyun cluster for users, and don't need to change session's code?